/**:
  ros__parameters:
    sock:
      buffer_size: 1024
      dst_hosts:
      - 127.0.0.1
      dst_ports:
      - 4321
      port: 1234

